exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/masked_modelling.ipynb

```diff
@@ -60,11 +60,11 @@
    "id": "e0ad672a-8b26-467b-8649-0bf95b1efb61",
```

Comment:
<p>, depending on the speed of the database connection.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/453/discussion/#comment-2589458071'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>